### DISCUSSION TITLE
mkdir() invalid argument

### DISCUSSION THREAD

**Discussion by ferdyhape** [misterdebug/crud-generator-laravel#5] [General]
i have error on create the crud process, error message is mkdir(): Invalid argument, i very happy if someone can help my error

**Comment by misterdebug** [comment]
hi @ferdyhape can you provide me complete error message pls ? And which command ? make:crud or api?
did you have publish vendor ? -> `php artisan vendor:publish --provider="Mrdebug\Crudgen\CrudgenServiceProvider"`

**Reply by gerardp** [reply]
You get that error if the Requests directory does not exist within the app/Http directory.

php artisan make:crud post "title:string, content:text"

**Comment by madfortech** [comment]
@misterdebug  can I customization by default resources posts view add  bootstrap or tailwind css classess ?

**Reply by misterdebug** [reply]
@madfortech I'm not sure to understand well your question but you can customize all the crud views files in your default-theme/layout but you can't customize classes generated by the crud (inputs in the html).

**Reply by madfortech** [reply]
![Capture](https://user-images.githubusercontent.com/21261716/212534024-40dfc70d-6c42-446c-aab8-5acb2d687a1e.PNG)

@extends('layouts.app')

how i input style change in posts/folder or 
resources/crudgen/views/default-theme/ change stub file ?

**Reply by misterdebug** [reply]
You can't customize the part between `Form::open` and `Form::close`, sorry
I can explain you where modify in the code if you want use your own fork. 
You can explain me what do you need. You need to change the class name `mb-3` only? Or more complex changes?

**Reply by madfortech** [reply]
style change input color border change  background ?

**Reply by misterdebug** [reply]
I think, it's specific needs but you can customize generated views files here : src/Services/MakeViewsService.php tell me if you are blocked :)

**Reply by madfortech** [reply]
no i am not blocked 
/****
	       <div class="mb-3">
			{{ Form::label('title', 'Title', ['class'=>'form-label ']) }}
			{{ Form::text('title', null, array('class' => 'form-control bg-success')) }}
		</div>
****/
that correct way customize file i add background color  input field ?

**Reply by misterdebug** [reply]
Yes it's a possible way to do this :)
